### PR TITLE
Add the Ciel

### PR DESCRIPTION
**Pull request by ramonimbao**
## Description

Adds the Ciel keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/chickenman/ciel/config.h [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/chickenman/ciel/config.h:105]: Deprecated, no longer exist.
```suggestion
```

File: keyboards/chickenman/ciel/info.json [hunk 2]
```diff
@@ -0,0 +1,219 @@
+{
+    "keyboard_name": "Ciel",
+    "url": "",
+    "maintainer": "ramonimbao",
+    "width": 15,
+    "height": 5,
```
Comment (drashna) [inline keyboards/chickenman/ciel/info.json:6]:
```suggestion
```

File: keyboards/chickenman/ciel/readme.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+The Ciel is a CNC'ed 60% Bakeneko with a brass internal weight.
+
+Group Buy ran October 7th to October 31st in 2021. Fulfilled early March.
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32u2
+* Hardware Availability: [GB page](https://geekhack.org/index.php?topic=114845.0), [ChickenMan's Discord](https://discord.gg/6MeZAxp2RP)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/chickenman/ciel/readme.md:13]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/chickenman/ciel/rules.mk [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/chickenman/ciel/rules.mk:21]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/chickenman/ciel/readme.md (hunk 3 above)
Comment (ramonimbao) [reply keyboards/chickenman/ciel/readme.md:13]: Added, thanks!

**Review by drashna — APPROVED** [review]

File: keyboards/chickenman/ciel/rules.mk [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u2
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no     # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/chickenman/ciel/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**